docs/compilers/CSharp/Compiler Breaking Changes - VS2017.md

```diff
@@ -1,6 +1,9 @@
-**This document lists known breaking changes in Roslyn 2.0 (VS2017) from Roslyn 1.\* (VS2015) and native C# compiler (VS2013 and previous).**
+**This document lists known breaking changes in Roslyn 2.0 *(VS2017*) from Roslyn 1\*. *(VS2015)* and native C# compiler *(VS2013 and previous)*.**
 
-*Breaks are formatted with a monotonically increasing numbered list to allow them to referenced via shorthand (i.e., "known break #1").
-Each entry should include a short description of the break, followed by either a link to the issue describing the full details of the break or the full details of the break inline.*
+<!--
```

Comment:
Please don't turn this into a comment. Same for the rest of the documents in this PR.

Reply:
For the most part, I think all but one document actually referenced this as a comment. Is it then more correct that this is intended for the reader as opposed to the task of compiling the list?

Reply:
I read it both ways: directive to the author, and and information for the reader.